db/migrate/20170901061225_create_wind_turbine_vibration_measurements.rb

```diff
@@ -0,0 +1,8 @@
+class CreateWindTurbineVibrationMeasurements < ActiveRecord::Migration[5.0]
```

Comment:
Speed is the only field of the WindTurbineSeedMeasurement model. Could we use only one model/table called WindTurbineMeasurement and include there all the fields related to the WindTurbine?